src/library/Uno.Material/Controls/BottomNavigationBar.cs

```diff
@@ -0,0 +1,122 @@
+﻿using System;
+using System.Collections.Generic;
+using Uno.Extensions.Specialized;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+
+namespace Uno.Material.Controls
+{
+	public partial class BottomNavigationBar : Control
+	{
+		public List<BottomNavigationBarItem> Items
+		{
+			get => (List<BottomNavigationBarItem>)GetValue(ItemsProperty);
+			set => SetValue(ItemsProperty, value);
+		}
+
+		public readonly DependencyProperty ItemsProperty =
+			DependencyProperty.Register(
+				"Items",
+				typeof(List<BottomNavigationBarItem>),
+				typeof(BottomNavigationBar),
+				new PropertyMetadata(new List<BottomNavigationBarItem>(), OnItemsChanged));
+
+		public BottomNavigationBarItem SelectedItem
+		{
+			get => (BottomNavigationBarItem)GetValue(SelectedItemProperty);
+			set => SetValue(SelectedItemProperty, value);
+		}
+
+		public static readonly DependencyProperty SelectedItemProperty =
+			DependencyProperty.Register(
+				"SelectedItem",
+				typeof(BottomNavigationBarItem),
+				typeof(BottomNavigationBar),
+				new PropertyMetadata(null, OnSelectedItemChanged));
+
+		public int SelectedIndex
```

Comment:
Do we really want SelectedIndex?

Reply:
I explained it to Mathieu, it covers more use cases